dripconfig/builtins.py

```diff
@@ -82,16 +83,33 @@ class StatsdConfig(SchemaBasedTrigger):
         'statsd': {
             Required('host'): basestring,
             Required('port', default=8125): Coerce(int),
+            Optional('sample_rate'): Coerce(int),
+            Optional('disabled'): bool,
```

Comment:
I'm not 100% certain of voluptuous's behavior, but I *think* ini files may play havoc with this if you don't use Coerce: 'True' and 'False' come in as strings. I could be remembering wrong but I have a vague memory of working around this.